CONTRIBUTING.md

```diff
@@ -26,6 +27,132 @@ The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
+## Hack On Libpod
+
+This section describe to you how to start to hack on libpod.
+
+### Prepare your environment
+
+Assume that you use Fedora 29, before starting to hack you need to install some requirements:
+```shell
+sudo dnf install -y \
+    podman \
+    atomic-registries \
+    btrfs-progs-devel \
+    conmon \
+    containernetworking-cni \
+    device-mapper-devel \
+    git \
+    glib2-devel \
+    glibc-devel \
+    glibc-static \
+    go \
+    golang-github-cpuguy83-go-md2man \
+    gpgme-devel \
+    iptables \
+    libassuan-devel \
+    libgpg-error-devel \
+    libseccomp-devel \
+    libselinux-devel \
+    make \
+    ostree-devel \
+    pkgconfig \
+    runc \
+    skopeo-containers
+```
+
+For [other distos you can find more available commands](install.md) on the install documentation to prepare your environment.
+
+Check that the installed version of `runc` is new enough by using:
+```shell
+$ runc --version
+runc --version
+runc version 1.0.0-rc6+dev
+commit: d164d9b08bf7fc96a931403507dd16bced11b865
+spec: 1.0.1-dev
+```
+
+You also need to check the installed version of golang, 1.10.x or higher is required:
+```shell
+$ go version
+go version go1.10.3 gccgo (GCC) 8.2.1 20181215 (Red Hat 8.2.1-6) linux/amd64
+```
+
+### Fork and clone libpod
+
+First you need to fork this project on github.
+
+Clone this project locally:
+```shell
+$ go get github.com/<you>/libpod
```

Comment:
I would not recommend using `go get` here. If somebody wants to contribute, then they need to `git clone` their fork on GitHub and ideally also `git remote add upstream ...` the main repository.

Reply:
agreed

Reply:
but `go get` work too and the git workflow was not impacted by this.